src/commons/editor/Editor.tsx

```diff
@@ -50,529 +64,249 @@ type StateProps = {
   breakpoints: string[];
   editorSessionId: string;
   editorValue: string;
-  highlightedLines: number[][];
+  highlightedLines: number[][]; // FIXME type this better??
```

Comment:
We probably should - can consider typing it as an array of `EditorSegment` - each a 2-element array specifying the starting and ending line indices of the segment of the program in the editor.

Reply:
Addressed in b5a249b.